wiki/Tournaments/GTS/ko.md

```diff
@@ -0,0 +1,35 @@
+# Global Taiko Showdown
+
+GTS 목록
```

Comment:
It should be translated to `Global Taiko Showdown 시리즈의 목차 페이지입니다.` directly, since other translated contest/tournament index pages is using the format: `[모든] <Full name of Tournament/Contest> 시리즈의 목차 페이지입니다.`